screen2.0/src/app/downloads/datamatrices.tsx

```diff
@@ -319,202 +372,241 @@ export function DataMatrices() {
+                      }}
+                      fullWidth
+                      >
+                      <MenuItem value="all">All</MenuItem>
+                      <MenuItem value="adult">Adult</MenuItem>
+                      <MenuItem value="embryonic">Embryonic</MenuItem>
+                    </Select>
+                  </Grid>
+                </Grid>
+              </Stack>
             </Stack>
-          </Grid>
-          <Grid size={12}>
-            {selectorButton({ assembly: "Human", assay: "DNase" })}
-            {selectorButton({ assembly: "Human", assay: "H3K4me3" })}
-            {selectorButton({ assembly: "Human", assay: "H3K27ac" })}
-            {selectorButton({ assembly: "Human", assay: "CTCF" })}
-          </Grid>
-          <Grid size={12}>
-            <Stack direction={"row"} justifyContent={"space-between"}>
-              <div>
-                <Typography mt="auto" variant="h5">
-                  Mouse
-                </Typography>
-                <Divider />
-                <Typography variant="subtitle2">926,843 cCREs</Typography>
-                <Typography variant="subtitle2">366 cell types</Typography>
-              </div>
-              <Image src={mouseTransparentIcon} alt={"Mouse Icon"} style={{maxWidth: '75px', maxHeight: '75px'}} />
+
+            {/* mouse section */}
+            <Stack direction="column" spacing={1}>
+              <Grid container direction="row" alignItems="flex-start" spacing={2}>
+                <Grid>
+                  <Image src={mouseTransparentIcon} alt={"Mouse Icon"} style={{ maxWidth: '75px', maxHeight: '75px' }} />
+                </Grid>
+                <Grid size="grow">
+                  <Stack direction="column" spacing={1}>
+                    <Typography variant="h5">Mouse</Typography>
+                    <Divider />
+                    <Stack direction="row" justifyContent="space-between">
+                      <Typography variant="subtitle2"><b>926,843</b> cCREs</Typography>
+                      <Typography variant="subtitle2"><b>366</b> cell types</Typography>
+                    </Stack>
+                  </Stack>
+                </Grid>
+              </Grid>
+              <Stack direction="row" spacing={2} alignItems="center">
+                {selectorButton({ assembly: "Mouse", assay: "DNase" })}
+                {selectorButton({ assembly: "Mouse", assay: "H3K4me3" })}
+                {selectorButton({ assembly: "Mouse", assay: "H3K27ac" })}
+                {selectorButton({ assembly: "Mouse", assay: "CTCF" })}
+              </Stack>
+              <Grid container justifyContent="flex-end">
+                <Grid size={{ xs: 5.75 }} mt={1}>
+                  <InputLabel sx={{ color: 'white' }} id="download-label">Download</InputLabel>
+                  <Button sx={{ height: '40px', lineHeight: '20px' }} size="medium" variant="contained" fullWidth endIcon={<Download />} onClick={handleOpenDownloadModal}>Download Data</Button>
```

Comment:
Add `sx={{textTransform: 'none'}}` to this. This should be defined in our theme but can just override this manually for now.